### PR TITLE
feat(phpmyadmin): Add `vip  db phpmyadmin` command

### PR DESCRIPTION
**Pull request by aswasif007**
## Description

In this PR, we are adding a new command `vip db phpmyadmin` which generates a URL to `phpmyadmin`, and opens in the default browser.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-db-phpmyadmin.js @6721.production`
1. Verify that it opened a URL with `/.wpvip/phpmyadmin/?token=...` prefix in your browser.

### CODE REVIEW

File: src/commands/phpmyadmin.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			exit.withError( 'Please specify an environment' );
+		}
+
+		this.log( 'Generating PhpMyAdmin URL...' );
+		const url = await generatePhpMyAdminAccess( this.env.id );
+
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-call
+		opn( url, { wait: false } );
+
+		this.log( 'Switch to your default browser.' );
```
Comment (saroshaga) [inline src/commands/phpmyadmin.ts]:
```suggestion
		this.log( `You can view your environment's data over at ${ url }` );
```

File: src/global.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+declare module 'opn';
```
Comment (abdullah-kasim) [inline src/global.d.ts]: See `<project-root>/types/copy-dir/index.d.ts` on how to add types for untyped node modules packages.

This is possible on this repo thanks to this line: https://github.com/Automattic/vip-cli/blob/4ccef58c0995d94c84017e403c6b16b0d4f107e9/tsconfig.json#L25-L25

**Comment by abdullah-kasim** [comment]
Hope you don't mind, I made a tiny bit of change to make all our lives easier in the typescript land.

File: src/global.d.ts (hunk 2 above)
Comment (abdullah-kasim) [reply src/global.d.ts]: Whoops, it turns out the types are available at `@types/opn@5.1.0` so we'll use that instead. Don't worry about the mismatched versions - types from `@types` are generally safe to bring in as long as the major versions are the same - can't remember where I read it, but that has also been my experience so far.

However if we need to bring the types in - we could always copy paste back from there.

**Comment by abdullah-kasim** [comment]
Types for graphql queries can be generated according to the instructions here: https://github.com/Automattic/vip-cli/blob/trunk/CONTRIBUTING.md#generating-the-types

File: src/commands/phpmyadmin.ts (hunk 1 above)
Comment (aswasif007) [reply src/commands/phpmyadmin.ts]: We probably don't want to print the `url` because it contains the access token.

File: src/global.d.ts (hunk 2 above)
Comment (saroshaga) [reply src/global.d.ts]: btw, just FYI, this package has been through a bit of a rollercoaster ride : `open` was deprecated in favour of `opn` and now `opn` is also been deprecated and `open` has been brought back to life 😂 cc @aswasif007

Comment (aswasif007) [reply src/global.d.ts]: 🎢 

Should we move to `open` and remove `opn` from VIP-CLI?

Comment (saroshaga) [reply src/global.d.ts]: We can do that separately

**Comment by aswasif007** [comment]
### 🔔 Note: Released with `@next` tag for testing: v2.36.0-dev.0 🔔

File: src/global.d.ts (hunk 2 above)
Comment (abdullah-kasim) [reply src/global.d.ts]: The new version of the package has started using ESM btw. 

And when it comes to ESM, trust me on this, but come back in 10 years before touching ESM. The tooling around it is pretty bad IMO.

**Comment by aswasif007** [comment]
### 🔔 Note: Released with @next tag for testing: v2.36.2-dev.0 🔔

**Comment by aswasif007** [comment]
### 🔔 Note: Released with `@next` tag for closed beta: `v2.37.0-dev.0` 🔔

**Comment by aswasif007** [comment]
### 🔔 Note: Released with `@next` tag for open beta: `v2.38.0-dev.0` 🔔

**Comment by aswasif007** [comment]
### 🔔 Note: Released with @next tag for open beta: v2.38.0-dev.1 🔔

**Review by saroshaga — APPROVED** [review]

File: src/commands/phpmyadmin.ts [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+				environmentId: envId,
+			},
+		},
+	} );
+
+	// Re-enable global error handling
+	enableGlobalGraphQLErrorHandling();
+
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+	return resp?.data?.generatePHPMyAdminAccess?.url as string;
```
Comment (sjinks) [inline src/commands/phpmyadmin.ts:97]: Isn't this the wrong member? Shouldn't it be `resp.data?.enablePHPMyAdmin?.success` and be a `Boolean`, not `string`?

Comment (aswasif007) [reply src/commands/phpmyadmin.ts:97]: I think you are right 🤔

**Pull request merged**